### ISSUE TITLE
TypeError: URL is not a constructor

### ISSUE THREAD

**Issue by chrisfeghali** [zuavra/nginx-ip-whitelister#29]
Haven't looked into it yet, but getting the following after the changes yesterday. Using the docker container btw.
```
TypeError: URL is not a constructor
    at Server.<anonymous> (file:///opt/nginx-iw/node_modules/crumbware/index.js:27:27)
    at Server.emit (node:events:518:28)
    at parserOnIncoming (node:_http_server:1137:12)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:119:17) Unhandled promise: Promise {
  <rejected> TypeError: URL is not a constructor
      at Server.<anonymous> (file:///opt/nginx-iw/node_modules/crumbware/index.js:27:27)
      at Server.emit (node:events:518:28)
      at parserOnIncoming (node:_http_server:1137:12)
      at HTTPParser.parserOnHeadersComplete (node:_http_common:119:17)
}
```

**Comment by zuavra** [comment]
Hi Christopher, thank you for raising this, my test environment was still on 1.4.x and wasn't testing 1.5.x at all. :sweat_smile:   Please upgrade to 1.5.1 (also caught another bug in the process). I've also published a docker image so you can use the `:1.5.1` or :`laters` image tags.

**Comment by chrisfeghali** [comment]
No worries! Thanks for the quick resolution 🙂
